app-backend/database/src/main/scala/com/azavea/rf/database/tables/ScenesToProjects.scala

```diff
@@ -111,8 +111,31 @@ object ScenesToProjects extends TableQuery(tag => new ScenesToProjects(tag)) wit
       )
     }
   }
+  
+  def setColorCorrectParamsBatch(projectId: UUID, params: BatchParams)(implicit database: DB) = {
+    val results = for {
```

Comment:
It's arguing over semantics, but the motivation for the endpoint was to have a single request that would fail if any of members failed, rather than independent requests for each member.  From an external perspective, DB errors are one source of failure but the count of affected records is another, and there's no obvious distinction, so the result of the set of actions isn't incidental.

Either way, I've read more of Slick's documentation and there are a couple other approaches I'll try that may get the desired behavior.